documentation/docs/examples/point-to-point-measurement.mdx

```diff
@@ -0,0 +1,159 @@
+---
+id: point-to-point-measurement
+title: Point to point measurement
+description: Demo shows how to measure point to point distance in Reveal.
+---
+
+import { DemoWrapper } from '@site/docs/components/DemoWrapper';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+The `MeasurementTool` is a measuring tool for point to point distance in 3D models, by clicking to- and from-points. The default measurement unit is meters, but the tool provides callbacks to allow configuring the unit.
+
+The tool provides ability to change measuring line width & color.
+
+:::note
+Recommended: `CameraControlsOptions.changeCameraTargetOnClick` should be set to `false` to avoid click event when `MeasurementTool` is active.
+:::
+
+<DemoWrapper name="Cognite3DViewerDemo" />
+
+## Point to point distance measurment
+To enter point-to-point measurement mode, use `MeasurementTool.enterMeasurementMode()`. This will register events for handling mouse clicks and touch events to allow placing the measurement end-points.
+
+```jsx runnable
+// import { MeasurementTool } from '@cognite/reveal/tools';
+
+const measurementTool = new MeasurementTool(viewer);
+measurementTool.enterMeasurementMode();
+```
+## Cancel measurement
+To exit from measurement mode, use `MeasurementTool.exitMeasurementMode()`.
+
+```jsx runnable
+// import { MeasurementTool } from '@cognite/reveal/tools';
+
+const measurementTool = new MeasurementTool(viewer);
+measurementTool.enterMeasurementMode();
+document.onkeydown = function(event) {
+  event = event || window.event;
+  if (event.keyCode == 27) {
+    measurementTool.exitMeasurementMode();
```

Comment:
`exitMeasurementMode()` doesn't remove the line being added before the camera is nudged, trigger `Cognite3dViewer.requestRedraw()`